cfn-resources/integration-tests/util/atlasHelper.go

```diff
@@ -0,0 +1,144 @@
+)
+
+var (
+	toolName           = cfnTool
+	userAgent          = fmt.Sprintf("%s/%s (%s;%s)", toolName, version.Version, runtime.GOOS, runtime.GOARCH)
+	terraformUserAgent = "terraform-provider-mongodbatlas"
+)
+
+func GetNewAtlasTeam(ctx context.Context, client *atlas.Client, name string, orgID string) (*atlas.Team, error) {
+	orgUser, _ := getExistingOrgUser(ctx, client, orgID)
+	teamRequest := atlas.Team{
+		Name:      name,
+		Usernames: []string{orgUser.Username},
+	}
+	team, _, err := client.Teams.Create(ctx, orgID, &teamRequest)
+	if err != nil {
+		log.Println("Error when creating Atlas Team")
+		return nil, err
+	}
+	return team, nil
+}
+
+func GetNewAtlasProject(ctx context.Context, client *atlas.Client, name string, orgID string) (*atlas.Project, error) {
+	project, _, err := client.Projects.Create(ctx, &atlas.Project{
+		Name:  name,
+		OrgID: orgID,
+	}, &atlas.CreateProjectOptions{})
+
+	if err != nil {
+		log.Println("Unable to create AWS client, please check AWS config is correctly setup")
+		return nil, err
+	}
+
+	return project, nil
+}
+
+func GetRealmClient(ctx context.Context) (*realm.Client, error) {
+	atlasEnv, err := getAtlasEnv()
+	if err != nil {
+		return nil, err
+	}
+	optsRealm := []realm.ClientOpt{realm.SetUserAgent(terraformUserAgent)}
+	authConfig := realmAuth.NewConfig(nil)
+	token, err := authConfig.NewTokenFromCredentials(ctx, atlasEnv.PublicKey, atlasEnv.PrivateKey)
+	if err != nil {
+		return nil, err
+	}
+
+	clientRealm := realmAuth.NewClient(realmAuth.BasicTokenSource(token))
+	realmClient, err := realm.New(clientRealm, optsRealm...)
+	if err != nil {
+		return nil, err
+	}
+
+	return realmClient, nil
+}
+
+func NewMongoDBClient() (atlasClient *atlas.Client, err error) {
+	atlasEnv, err := getAtlasEnv()
+	if err != nil {
+		return nil, err
+	}
+	t := digest.NewTransport(atlasEnv.PublicKey, atlasEnv.PrivateKey)
+	client, _ := t.Client()
+
+	opts := []atlas.ClientOpt{atlas.SetUserAgent(userAgent)}
+	if baseURL := atlasEnv.BaseURL; baseURL != "" {
+		opts = append(opts, atlas.SetBaseURL(baseURL))
+	}
+
+	mongodbClient, err := atlas.New(client, opts...)
+	if err != nil {
+		return nil, errors.New("unable to create Atlas client")
+	}
+
+	return mongodbClient, nil
+}
+
+func GetNewBasicClusterWithSampleData(ctx context.Context, client *atlas.Client, projectID string) (*atlas.AdvancedCluster, error) {
+	// TODO: implement
+	return nil, nil
+}
+
+func getAtlasEnv() (atlasEnvOpts *AtlasEnvOptions, err error) {
+	orgID, OrgIDOk := os.LookupEnv("ATLAS_ORG_ID")
+	publicKey, publicKeyOk := os.LookupEnv("ATLAS_PUBLIC_KEY")
+	privateKey, privateKeyOk := os.LookupEnv("ATLAS_PRIVATE_KEY")
+	baseURL, baseURLOk := os.LookupEnv("ATLAS_BASE_URL")
+
+	if !OrgIDOk || !privateKeyOk || !publicKeyOk || !baseURLOk {
+		return nil, errors.New("please ensure following env variables are set: " +
+			"ATLAS_ORG_ID, ATLAS_PUBLIC_KEY, ATLAS_PRIVATE_KEY, ATLAS_BASE_URL, ATLAS_SECRET_PROFILE")
+	}
```

Comment:
FYI 

```bash
ATLAS_ORG_ID=''
```
will return `OrgIDOk == true` you don't need `LookupEnv`, plain `os.Getenv` works and you need to make sure `orgID != ""`

Reply:
updated